dropwizard-logging/src/main/java/io/dropwizard/logging/DefaultLoggingFactory.java

```diff
@@ -170,10 +192,24 @@ private Logger configureLevels() {
 
         root.setLevel(level);
 
-        for (Map.Entry<String, Level> entry : loggers.entrySet()) {
-            loggerContext.getLogger(entry.getKey()).setLevel(entry.getValue());
-        }
+        if (advancedLoggers != null && advancedLoggers.size() > 0) {
```

Comment:
Does this mean that if I define a single advanced logger that all the other loggers won't take affect?

Given the following configuration

```yaml
level: INFO
loggers:
  "com.example.dw.Thing": DEBUG
advancedLoggers:
  "com.example.app": 
    level: DEBUG
    appenders:
      - type: console
        threshold: ALL
```

Correct me if I'm wrong but the logic here makes me think that `com.example.dw.Thing` won't be registered as `DEBUG`